src/containers/Page.jsx

```diff
@@ -84,20 +91,29 @@ class Page extends Component {
 
   render() {
     const { userInfo } = this.props;
+    const society = {
```

Comment:
@joykare Since redux isn't hooked up yet, I am passing this in as props. This will be removed and the data obtained from the store when we implement the redux. Same for the society stats, and activities. To see the logos for the respective societies change the name property for this object e.g to sparks, istelle or phoenix

Reply:
I thought @john555 had done a task already where he implemented redux.

Reply:
Yeah, Redux has been setup. What I meant is that view is based on the data for the society fetched from api when the page loads. Since we do not yet have an endpoint to fetch data from, I am passing in this object as props to represent society data